src/concepts/what-is-zilla.md

```diff
@@ -0,0 +1,132 @@
+
+## Zilla Features
+
+### Self-Contained, Stateless Architecture
+
+Zilla has no external dependencies, is stateless, and is highly memory efficient. When deployed as an edge proxy, it scales horizontally to support millions of concurrently connected clients.
+
+### HTTP/2, gRPC, SSE, MQTT, and Kafka Support + Protocol Mediation
+
+Inside Zilla, every protocol, whether it is TCP, TLS, HTTP, Kafka, gRPC, etc., is treated as a stream, so mediating between protocols simplifies to mapping protocol-specific metadata.
+
+### Protobuf, Avro, and JSON Schema payloads
+
+Zilla supports payload schema specifications for message validation and translation.
+
+### OpenAPI and AsyncAPI Support
+
+Zilla supports different API schema specifications for message validation and API creation.
+
+### API Registry and Schema Registry Integrations
+
+Zilla integrates with schema registries and API registries including Apicurio, Confluent Schema Registry (Zilla Plus), and Karapace.
+
+### Security
+
+Zilla can terminate TLS and supports JWT-based authorization for REST, SSE, and MQTT endpoints/services.
+
+### Observability
+
+Zilla supports OpenTelemetry for metrics and logging and can expose a Prometheus metrics endpoint. Syslog for logging is supported in Zilla Plus.
+
+## Zilla Use Cases
+
+Zilla can be used as a service proxy (sidecar) or as an AsyncAPI Kafka gateway.
+
+### Service Proxy
+
+When deployed in front of an existing HTTP, SSE (Server Sent Events), MQTT, Kafka, or gRPC service, Zilla adds metrics, logging, message validation, and authentication.
+
+#### For HTTP Services
+
+- Gather metrics and telemetry data on traffic flowing in and out of an HTTP service.
+- Add JWT-based client authentication.
+- Enforce OpenAPI schema definitions and reject invalid requests.
+
+#### For SSE Services
+
+- Gather metrics and telemetry data on traffic flowing out of an SSE service.
+- Add JWT-based client authentication with [Continuous Authorization](https://www.aklivity.io/post/a-primer-on-server-sent-events-sse#:~:text=Securing%20SSE%20with%20aklivity%20Zilla). Zilla supports Continuous Authorization which gracefully re-authorizes a client on an SSE server’s behalf, without abruptly terminating message streams.
+
+#### For MQTT Services
+
+- Gather metrics and telemetry data on traffic flowing in and out of an MQTT service.
+- Add JWT-based client authentication.
+- Enforce AsyncAPI schema definitions and reject invalid inbound messages.
+
+#### For Kafka Services
+
+- Gather metrics and telemetry data on traffic flowing in and out of a Kafka service.
+- Enforce AsyncAPI schema definitions and reject invalid inbound messages.
+
```

Comment:
We don't have the `kafka` server binding kind yet, so we cannot do `kafka` server to `kafka` client back-to-back like the other protocols just yet.